.s2i/bin/assemble

```diff
@@ -2,7 +2,7 @@
 
 set -x
 
-set -eo pipefail
+# set -eo pipefail
```

Comment:
we can unset this, as the ci issue is fixed with issue #7

Reply:
@akchinSTC Please revert the debug changes:)